core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.logging;
```

Comment:
internal

Reply:
If I apply the change, LoggingDecoratorBuilder's internal public methods will be exposed to public. (and causes build error)
```
* Where:
Build file '/Users/masa/src/armeria/site/build.gradle' line: 131

* What went wrong:
Execution failed for task ':site:test'.
> java.lang.Exception: Blacklisted class(es) in the public API:
  - LoggingClientBuilder -> com.linecorp.armeria.internal.logging.LoggingDecoratorBuilder
  - LoggingServiceBuilder -> com.linecorp.armeria.internal.logging.LoggingDecoratorBuilder
```

Should I add this class to whitelist?

Reply:
any idea for how to fix this? @anuraaga, @trustin

Reply:
Oops, sorry for being late. I think you should put this into `common.logging`. Otherwise our public API will expose a class from the internal package in the class hierarchy. I think it's fine to keep this class in `common.logging` therefore.